ocm-container.sh

```diff
@@ -143,7 +143,7 @@ ${PAGERDUTYFILEMOUNT} \
 ${AWSFILEMOUNT} \
 ${SSH_AGENT_MOUNT} \
 ${OPS_UTILS_DIR_MOUNT} \
--v /root/.ssh/sockets \
+-v ${HOME}/.ssh/sockets:/root/.ssh/sockets \
```

Comment:
this means it's read-write, not sure if we want to open this be default 😨

Reply:
That anything read write is not on by default I ocm-container
And allowing this allows for data to leak without thinking

Reply:
i'm not sure i follow - what data leak would be possible from mounting it read-write instead of read-only?

Reply:
Data can be saved there and read from another container.
Someone can craft a problematic file that will cause issues while we login via backplane

This tool is supposed to be ephemeral and not write anything back for persistence.

This is why I added this file.

That being said, I am stepping down from being a maintainer soon, so you can discuss in our internal channels

Reply:
requiring pin+token every time we start an ocm-container is a huge pain and makes me not want to use ocm-container at all, honestly

since "an SRE could (intentionally or otherwise) craft a malicious file with data from one cluster and have it cause problems on another cluster" is also a risk for using `ocm backplane` normally (ie: outside of a container) i feel like this is an acceptable security/convenience tradeoff

Reply:
Ok, I hope this will be the only leak we create.